src/Microsoft.AspNetCore.Mvc.Razor.Host/Internal/ViewComponentTagHelperChunkVisitor.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Chunks;
+using Microsoft.AspNetCore.Razor.CodeGenerators;
+using Microsoft.AspNetCore.Razor.CodeGenerators.Visitors;
+using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Host.Internal
+{
+    /// <summary>
+    /// A <see cref="CodeVisitor{CSharpCodeWriter}"/> that parses <see cref="Chunk"/>s and 
+    /// generates code based on the contents of the chunks that are view component tag helpers.
+    /// </summary>
+    public class ViewComponentTagHelperChunkVisitor : CodeVisitor<CSharpCodeWriter>
+    {
+        private GeneratedViewComponentTagHelperContext _context;
```

Comment:
Don't see the `GeneratedViewComponentTagHelperContext` in the list of files/changes.